notify_credentials.go

```diff
@@ -0,0 +1,86 @@
+package twilio
+
+import (
+	"context"
+	"net/url"
+)
+
+type CredentialType string
+
+const credentialsPathPart = "Credentials"
+
+const (
+	TypeGCM = CredentialType("gcm")
+	TypeFCM = CredentialType("fcm")
+	TypeAPN = CredentialType("apn")
+)
+
+type NotifyCredentialsService struct {
+	client *Client
+}
+
+type NotifyCredential struct {
+	Sid          string         `json:"sid"`
+	FriendlyName string         `json:"friendly_name"`
+	AccountSid   string         `json:"account_sid"`
+	Type         CredentialType `json:"type"`
+	DateCreated  TwilioTime     `json:"date_created"`
+	DateUpdated  TwilioTime     `json:"date_updated"`
+	URL          string         `json:"url"`
+}
+
+type NotifyCredentialPage struct {
+	Page
+	Credentials []*NotifyCredential `json:"credentials"`
+}
+
+type NotifyCredentialPageIterator struct {
+	p *PageIterator
+}
+
+func (n *NotifyCredentialsService) Create(ctx context.Context, data url.Values) (*NotifyCredential, error) {
```

Comment:
Would it make sense to add a helper here, where we initialize some defaults, a la `Messages.SendMessage`?